### DISCUSSION TITLE
deta base storage size

### DISCUSSION THREAD

**Discussion by IndrajeethY** [deta/cloud-docs#524] [Help]
is it possible to know about current deta base size which i've stored data?

**Comment by lemonyte** [comment]
There is no counter in the UI, but you can write a little script to download all the data and save it to a JSON. The file size would give you a general approximation of the Base's size.

```python
import json
from deta import Deta

deta = Deta()
db = deta.Base("mydb")

response = db.fetch()
all_items = response.items
while response.last:
    response = db.fetch(last=response.last)
    all_items.extend(response.items)

with open("mydb.json", "w") as file:
    json.dump(all_items, file)
